invenio_communities/templates/semantic-ui/invenio_communities/details/macros/detail.html

```diff
@@ -0,0 +1,15 @@
+{#
+    Copyright (C) 2022 CERN.
+  
+    Invenio RDM Records is free software; you can redistribute it and/or modify
+    it under the terms of the MIT License; see LICENSE file for more details.
+  #}
+
+{% macro truncate_text(text) %}
+  {% if text|length > 216 %}
```

Comment:
why is there a difference, here you check if it has 216 length and below you truncate to 211? 
How did you come up with both numbers?

Reply:
The discrepancy between 211 and 216 was established during testing. Even passing the killwords parameter as true in jinja (see https://jinja.palletsprojects.com/en/3.0.x/templates/#jinja-filters.truncate), we can find a discrepancy of 5 characters between the breaking point in which the filter truncates the text and the passed parameter. I didn't check the code behind the jinja template, but probably due to it considering the " ... " characters. 

About the discrepancy between 216 and 225, it's because "Show all" or "Show less" take 9 characters.

Reply:
but I am still not sure how did you come up with those numbers?
in any case I think it will have problems on different screen resolutions, the numbers should be also unified.